### DISCUSSION TITLE
Has anyone gotten <leader>tT (run all test files) to work?

### DISCUSSION THREAD

**Discussion by fredrikaverpil** [LazyVim/LazyVim#901] [Q&A] [answered]
I'm using neotest-python and I can see that the LazyVim docs [specifies this](https://www.lazyvim.org/plugins/extras/test.core#neotest-1) to run on `<leader>tT`:

```lua
require("neotest").run.run(vim.loop.cwd())
```

My [neotest adapter config](https://github.com/fredrikaverpil/dotfiles/blob/main/nvim-lazyvim/lua/plugins/test.lua#L34) specifies the pytest runner like this:

```lua
adapters = {
  require("neotest-python")({
    runner = "pytest",
    args = { "--log-level", "INFO", "--color", "yes", "-vv", "-s" },
  }),
```

...but I don't see any tests running in the summary (`<leader>ts`) or in the output window (`<leader>tO`) so I am guessing this doesn't quite work as intended.

Anyone gotten this to work - and/or can see what I'm doing wrong here?

**Comment by folke** [comment, marked as answer]
Works for me with plenary lua tests.

**Reply by fredrikaverpil** [reply]
Okay, so I should likely take this to [neotest-python](https://github.com/nvim-neotest/neotest-python) and ask there. Thanks!

**Reply by fredrikaverpil** [reply]
For anyone else who might find this discussion, I opened up an issue for a potential improvement #931
